### ISSUE TITLE
For React only?

### ISSUE THREAD

**Issue by jkleiser** [pedronauck/reworm#16]
From the readme I get the impression that Reworm is for use with React projects only. Has it been successfully tested with alternatives like Angular or Vue?

**Comment by pietrofxq** [comment]
This library is not tied to react at all. You only call functions to set and get state, so it should work with anything

**Comment by auwtch** [comment]
yep, reworm's get function returns a react specific Consumer component that handles re-render logic.

**Comment by cowboyd** [comment]
@jkleiser You can try microstates. A similar API which works with React, Angular, Vue, and Ember. https://github.com/microstates/microstates.js
